containers/dotnetcore-mssql/.devcontainer/library-scripts/node-debian.sh

```diff
@@ -0,0 +1,119 @@
+#!/bin/bash
```

Comment:
You actually don't need this file since you aren't building `base.Dockerfile`.  You just will want `library-scripts\azcli-debian.sh`